.github/workflows/test_full.yml

```diff
@@ -39,16 +39,16 @@ jobs:
       - name: install dependencies
         run: |
           python -m pip install --upgrade pip
-          python -m pip install -e .[test_full]
+          python -m pip install -e .[devel_full]
         shell: bash
 
       - name: run pytest
         run: |
           if [ "$RUNNER_OS" == "Linux" ]; then
               source <(cat .env_file | sed 's/=/=/' | sed 's/^/export /')
-              pytest --tb=long -vv --cache-clear --no-cov --allow-skip=telegram tests/
+              pytest -m "not slow" --tb=long -vv --cache-clear --no-cov --allow-skip=telegram tests/
           else
-              pytest --tb=long -vv --cache-clear --no-cov --allow-skip=telegram,docker tests/
+              pytest -m "not slow" --tb=long -vv --cache-clear --no-cov --allow-skip=telegram,docker tests/
```

Comment:
I would expect `test_full` job to run **all** tests, not **all but for slow ones**.

Reply:
I didn't want to include the only slow test into these because the test was about checking tutorial dependencies.
But I guess it would be good to check that tutorials run successfully on different python versions and different OSs.

Removed the limitation.